src/xcode/ENA/ENA/Source/Scenes/FooterViewController/FooterViewController.swift

```diff
@@ -60,26 +56,113 @@ class FooterViewController: UIViewController {
 		view.addSubview(primaryButton)
 		view.addSubview(secondaryButton)
 
+		primaryButton.hasBackground = true
+		primaryButton.addTarget(self, action: #selector(didHitPrimaryButton), for: .primaryActionTriggered)
 		primaryButton.translatesAutoresizingMaskIntoConstraints = false
+		
+		secondaryButton.hasBackground = true
+		secondaryButton.addTarget(self, action: #selector(didHitSecondaryButton), for: .primaryActionTriggered)
 		secondaryButton.translatesAutoresizingMaskIntoConstraints = false
 
-		NSLayoutConstraint.activate([
+		updateViewModel()
+	}
+
+	// MARK: - Internal
+
+	var viewModel: FooterViewModel {
+		didSet {
+			updateViewModel()
```

Comment:
@KaiTeuber I added the ability to update the footer view model. I needed this because in one of the view controllers the buttons would change depending on a state.